packages/core/src/plugins/PluginManager.ts

```diff
@@ -55,8 +73,6 @@ class PluginManager {
    * Load all plugins of the site
    */
   _collectPlugins() {
-    module.paths.push(path.join(this.config.rootPath, 'node_modules'));
```

Comment:
Removed this line because the `module` object is not compatible with ES6 imports. This is manually implemented below around line 146.